src/sentry/rules/processing/delayed_processing.py

```diff
@@ -255,10 +253,11 @@ def process_delayed_alert_conditions(buffer: RedisBuffer) -> None:
     time_limit=60,  # 1 minute
     silo_mode=SiloMode.REGION,
 )
-def apply_delayed(project_id: int) -> None:
+def apply_delayed(project_id: int, date_added: float) -> None:
```

Comment:
Let's add `*args, **kwargs` to the definition as well. This allows us to add/remove fields here without breaking things during deploy